source/uwp/Renderer/lib/AdaptiveImageRenderer.cpp

```diff
@@ -666,4 +744,93 @@ namespace AdaptiveCards::Rendering::Uwp
         auto foundSvg = url.find("svg");
         return !(foundSvg == std::string::npos);
     }
+
+    void render_xaml::XamlBuilder::ParseXmlForHeightAndWidth(winrt::XmlDocument const& xmlDoc,
+                                                             winrt::SvgImageSource const& imageSource)
+    {
+        if (xmlDoc)
+        {
+            auto rootElement = xmlDoc.DocumentElement();
+            auto height = rootElement.GetAttribute(L"height");
+            auto width = rootElement.GetAttribute(L"width");
+
+            // We only need to set height or width, not both (fixes aspect ratio for person style)
+            bool isHeightSet = false;
+
+            if (!height.empty())
+            {
+                if (auto heightAsDouble = HStringToDouble(height))
+                {
+                    SetRasterizedPixelHeight(imageSource, heightAsDouble);
+                    isHeightSet = true;
+                }
+            }
+
+            if (!width.empty())
+            {
+                if (auto widthAsDouble = HStringToDouble(width))
+                {
+                    SetRasterizedPixelWidth(imageSource, widthAsDouble, isHeightSet);
+                }
+            }
+        }
+    }
+
+    winrt::IAsyncAction render_xaml::XamlBuilder::SetRasterizedPixelHeight(winrt::SvgImageSource const imageSource,
+                                                                           double imageSize,
+                                                                           bool dropIfUnset)
+    {
+        co_await winrt::resume_foreground(imageSource.Dispatcher());
+        auto currentSize = imageSource.RasterizePixelHeight();
+        bool sizeIsUnset = isinf(currentSize);
+
+        bool dropHeight = sizeIsUnset && dropIfUnset;
```

Comment:
>bool [](http://example.com/codeflow?start=8&length=4)

[nit] Many variables in this method and the nearby methods could be marked as `const`.